account_asset_management/views/account_move.xml

```diff
@@ -45,8 +45,13 @@
                 <field
                     name="asset_profile_id"
                     domain="[('company_id','=', parent.company_id)]"
+                    optional="show"
```

Comment:
I think this one should be `hide`. Most of the time you are not entering asset invoices.

Reply:
ok, done